template/web/app/paymaster-bundler/constants.ts

```diff
@@ -0,0 +1,29 @@
+export const rpcUrl =
+  'https://api.developer.coinbase.com/rpc/v1/base-sepolia/Nz9nSdr8uIY7HhIi2zRrJFgISslFF2In';
```

Comment:
Should this be passed via process.env.RPC_URL instead?

Reply:
done in new commit